### PR TITLE
VACMS-5237 Update javascript deps on vagovadmin theme

### PR DESCRIPTION
**Pull request by indytechcook**
## Description

See #5237. 

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Review by acrollet — DISMISSED** [review]

**Comment by indytechcook** [comment]
looks like npm audit downgraded a bunch of code for some reason.

**Comment by indytechcook** [comment]
Validation is on the linked issue #5237.

This PR also includes remove the npm way of doing things.

File: docroot/themes/custom/vagovadmin/package.json [hunk 1]
```diff
@@ -2,29 +2,23 @@
...
     "browser-sync": "^2.26.14",
     "child_process": "^1.0.2",
-    "cssnano": "^5.0.0-rc.2",
     "gulp": "^4.0.2",
     "gulp-autoprefixer": "*",
     "gulp-babel": "^8.0.0",
     "gulp-concat": "^2.6.1",
     "gulp-sass": "4.1.0",
-    "gulp-sourcemaps": "^3.0.0",
+    "gulp-sourcemaps": "^2.6.5",
```
Comment (acrollet) [inline docroot/themes/custom/vagovadmin/package.json:20]: hmm, what's the reason for the downgrade in the sourcemaps packages?

Comment (indytechcook) [reply docroot/themes/custom/vagovadmin/package.json:20]: Not sure.  I ran `npm audit fix` and that's what it did.  I'll put it back

File: docroot/themes/custom/vagovadmin/assets/css/styles.min.css [hunk 2]
```diff
@@ -1,2 +1,2 @@
...s-0-subform-field-non-beneficiares-wrapper],.field--name-field-clp-faq-paragraphs [id^=edit-field-tags-0-subform-field-non-beneficiares-wrapper],.field--name-field-primary-call-to-action [id^=edit-field-tags-0-subform-field-non-beneficiares-wrapper]{display:none}.node-campaign-landing-page-edit-form summary,.node-campaign-landing-page-form summary{text-transform:none}.visual-layout-svg{padding:30px 30px 55px 0}.field--name-field-facility-service-hours .seven-details,.field--name-field-facility-service-hours .tablefield-addrow.button,.field--name-field-service-location .paragraph-type-title{display:none}.tooltip-layout{border:1px solid silver;border-radius:2px;margin:1em 0;min-width:0;padding:30px 18px 18px;position:relative}.tooltip-toggle{background-image:url(../../images/icons/questionmark-disc-000000.svg);background-size:cover;border:0;height:25px;position:absolute;right:10px;top:10px;width:25px}.tooltip-layout [id^=add-tooltip-]{display:none}.centralized,.not-editable{background-color:#edede8!important}.claro-spinner{background-image:url(../../../../../core/themes/claro/images/spinner-ltr.gif)}#edit-group-facility-api .field--name-field-facility-hours input{pointer-events:none}
```
Comment (acrollet) [inline docroot/themes/custom/vagovadmin/assets/css/styles.min.css:1]: sorry, one more question - are the minified files still generated with CSS? if not, should we just remove them from VC?

Comment (indytechcook) [reply docroot/themes/custom/vagovadmin/assets/css/styles.min.css:1]: Yes, it was part of the gulp task.

Comment (indytechcook) [reply docroot/themes/custom/vagovadmin/assets/css/styles.min.css:1]: When I do anything other than ['compress' as the option](https://github.com/sass/node-sass#options), it creates a new file per sass file:

```
❯ git status
On branch VACMS-5237_vagov_admin_deps
Your branch is up to date with 'origin/VACMS-5237_vagov_admin_deps'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   assets/css/proofing.css
	modified:   assets/css/proofing.css.map
	modified:   assets/css/styles.css
	modified:   assets/css/styles.css.map
	modified:   assets/css/user_guides.css
	modified:   assets/css/user_guides.css.map
	modified:   assets/css/whats_new.css
	modified:   assets/css/whats_new.css.map
	modified:   assets/css/wysiwyg.css
	modified:   assets/css/wysiwyg.css.map
	modified:   gulpfile.js
```

**Review by acrollet — APPROVED** [review]

**Pull request merged**